chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -48,14 +62,15 @@ def forward_cpu(self, inputs):
         if count == 0:
             return numpy.zeros((), dtype=x.dtype),
 
-        y = (numpy.log(p) * (t.flat != self.ignore_label)).sum(keepdims=True) \
+        y = (log_p * (t.flat != self.ignore_label)).sum(keepdims=True) \
             * (-1.0 / count)
```

Comment:
I tried to raise an error with `count == 65536`, but it causes no error.
 `count` is scalar, isn't it?